### PR TITLE
K8SPG-46 update readme

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/41/commits/d45860c2b329e2f2ce5d62c833cd7977b7d244b9
image: `perconalab/percona-postgresql-operator:PR-41-d45860c2-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-41-d45860c2-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-41-d45860c2-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-41-d45860c2-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-41-d45860c2-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-41-d45860c2-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/41/commits/f101b7238516410a64c90257a69d09eb7b5e8c93
image: `perconalab/percona-postgresql-operator:PR-41-f101b723-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-41-f101b723-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-41-f101b723-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-41-f101b723-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-41-f101b723-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-41-f101b723-pgo-deployer`

File: README.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+* **High Availability**
+
+    Safe, automated failover backed by a distributed consensus based high-availability solution. Uses Pod Anti-Affinity to help resiliency; you can configure how much enforced this can be. Failed primaries automatically heal, allowing for faster recovery time. Support for standby PostgreSQL clusters that work both within and across multiple Kubernetes clusters.
+
+* **Disaster Recovery**
+
+    Backups and restores leverage the open source pgBackRest utility and includes support for full, incremental, and differential backups as well as efficient delta restores. Set how long you want your backups retained for. Works great with very large databases!
+
+* **TLS**
```
Comment (LenzGr) [inline README.md:23]: "TLS" is how secure communication is facilitated between nodes. I'd suggest to rather focus on the functionality here:
```suggestion
* **Communication Security**
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+* **Monitoring**
+
+    Track the health of your PostgreSQL clusters with Percona Monitoring and Management (PMM).
+
+* **PostgreSQL User Management**
+
+    Quickly add and remove users from your PostgreSQL clusters with powerful commands. Manage password expiration policies or use your preferred PostgreSQL authentication scheme.
+
+* **Upgrade Management**
```
Comment (LenzGr) [inline README.md:35]: Update or Upgrade? The latter one is usually used in the context of moving to a new major version.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/41/commits/5e2cda57b0ad828b0b99c7e8694a4127bb851a12
image: `perconalab/percona-postgresql-operator:PR-41-5e2cda57-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-41-5e2cda57-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-41-5e2cda57-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-41-5e2cda57-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-41-5e2cda57-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-41-5e2cda57-pgo-deployer`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+The variety of different Storage Classes available for Kubernetes is too wide to verify the Operator functionality in each one. With that said, the Operator is designed to be storage class agnostic and has been demonstrated to work with additional Storage Classes.
+
+## Installation
+
+The following steps will quickly make the Operator up and running with cloud native PostgreSQL:
+
+1. Clone the percona-postgresql-operator repository:
+
+    ```
+    $ git clone /https://github.com/percona/percona-postgresql-operator
```
Comment (spron-in) [inline README.md:112]: for now it is main branch?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+The Operator includes various components that get deployed to your Kubernetes cluster as shown in the following diagram and detailed in the Design section of the documentation for the version you are running.
+
+The Operator is developed and tested on CentOS and RHEL platforms but is known to run on other Linux variants.
+
+### Supported Platforms
+
+The Operator is tested against the following platforms:
+
+* Kubernetes 1.17+
```
Comment (spron-in) [inline README.md:94]: is it correct @hors ? Shall we keep only GKE and EKS?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+    * Selecting different storage classes for your primary, replica, and backup storage
+    * Select your own container resources class for each PostgreSQL cluster deployment; differentiate between resources applied for primary and replica clusters
+    * Use your own container image repository, including `imagePullSecrets` and private repositories  support
+    * Customize your PostgreSQL configuration 
+    * Bring your own trusted certificate authority (CA) for use with the Operator API server
+    * Override your PostgreSQL configuration for each cluster
+
+## Deployment Requirements
+
+The Operator is validated for deployment on Kubernetes, GKE and EKS clusters. the Operator is cloud native and storage agnostic, working with a wide variety of storage classes, hostPath, and NFS.
```
Comment (spron-in) [inline README.md:84]:
```suggestion
The Operator is validated for deployment on Kubernetes, GKE and EKS clusters. The Operator is cloud native and storage agnostic, working with a wide variety of storage classes, hostPath, and NFS.
```

**Review by spron-in — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (hors) [reply README.md:112]: for now it should be 'release-0.1.0' , but after the release it will be v0.1.0 tag

File: README.md (hunk 4 above)
Comment (hors) [reply README.md:94]: yes, we need to keep only GKE and EKS

File: README.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+* **Full Customizability**
+
+    The Operator not only makes it easy to get PostgreSQL up and running on Kubernetes-enabled platforms, but also allows you to further customize your deployments, including:
+
+    * Selecting different storage classes for your primary, replica, and backup storage
+    * Select your own container resources class for each PostgreSQL cluster deployment; differentiate between resources applied for primary and replica clusters
+    * Use your own container image repository, including `imagePullSecrets` and private repositories  support
+    * Customize your PostgreSQL configuration 
+    * Bring your own trusted certificate authority (CA) for use with the Operator API server
+    * Override your PostgreSQL configuration for each cluster
```
Comment (hors) [inline README.md:80]: Please add additional point, something like:   'Use your own custom images, you can re-define the image for each container separately'

File: README.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+
+## Deployment Requirements
+
+The Operator is validated for deployment on Kubernetes, GKE and EKS clusters. The Operator is cloud native and storage agnostic, working with a wide variety of storage classes, hostPath, and NFS.
+
+The Operator includes various components that get deployed to your Kubernetes cluster as shown in the following diagram and detailed in the Design section of the documentation for the version you are running.
+
+The Operator is developed and tested on CentOS and RHEL platforms but is known to run on other Linux variants.
```
Comment (hors) [inline README.md:88]: We use 'Red Hat Universal Base Image 8 Minimal (UBI 8)' for all images.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+The following steps will quickly make the Operator up and running with cloud native PostgreSQL:
+
+1. Clone the percona-postgresql-operator repository:
+
+    ```
+    $ git clone /https://github.com/percona/percona-postgresql-operator
+    $ cd percona-postgresql-operator
+    ```
+
+2. Deploy the operator with the following command:
```
Comment (hors) [inline README.md:116]: you need to create pgo namespace, the point 2 should be` kubectl create namespace pgo `

**Review by hors — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/41/commits/cbf9e8ce5739a3b9c2acc96e3c2ca6a7bc7ba500
image: `perconalab/percona-postgresql-operator:PR-41-cbf9e8ce-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-41-cbf9e8ce-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-41-cbf9e8ce-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-41-cbf9e8ce-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-41-cbf9e8ce-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-41-cbf9e8ce-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/41/commits/6e776e5ef0bda655b102ee0627a4c02ae91daa7b
image: `perconalab/percona-postgresql-operator:PR-41-6e776e5e-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-41-6e776e5e-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-41-6e776e5e-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-41-6e776e5e-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-41-6e776e5e-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-41-6e776e5e-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/41/commits/bc71b5a4e37ecc6d98a089af6d415d29c4a0d6ea
image: `perconalab/percona-postgresql-operator:PR-41-bc71b5a4-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-41-bc71b5a4-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-41-bc71b5a4-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-41-bc71b5a4-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-41-bc71b5a4-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-41-bc71b5a4-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/41/commits/086bd18e8585bd7db014f0bb5bfc1bbf73c38727
image: `perconalab/percona-postgresql-operator:PR-41-086bd18e-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-41-086bd18e-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-41-086bd18e-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-41-086bd18e-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-41-086bd18e-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-41-086bd18e-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/41/commits/6c7e4d71c3ff3b6a6b62c49639fc4076178f9f39
image: `perconalab/percona-postgresql-operator:PR-41-6c7e4d71-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-41-6c7e4d71-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-41-6c7e4d71-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-41-6c7e4d71-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-41-6c7e4d71-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-41-6c7e4d71-pgo-deployer`

File: README.md [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+* [PostgreSQL](https://www.postgresql.org/)
+  * [PostgreSQL Contrib Modules](https://www.postgresql.org/docs/current/contrib.html)
+  * [pgAudit](https://www.pgaudit.org/)
+  * [set_user](https://github.com/pgaudit/set_user)
+  * [wal2json](https://github.com/eulerto/wal2json)
+* [pgBackRest](https://pgbackrest.org/)
+* [pgBouncer](http://pgbouncer.github.io/)
+* [Patroni](https://patroni.readthedocs.io/)
+* LLVM (for JIT compilation)
+
```
Comment (hors) [inline README.md:153]: we need to mention that we also use pg_stat_monitor 
https://github.com/percona/pg_stat_monitor/

**Review by hors — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| scaling | passed |
| recreate | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/41/commits/fdcad7518b48eda966c45e1d63e4a2f43923420b
image: `perconalab/percona-postgresql-operator:PR-41-fdcad751-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-41-fdcad751-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-41-fdcad751-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-41-fdcad751-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-41-fdcad751-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-41-fdcad751-pgo-deployer`

**Pull request merged**